Userland/Libraries/LibGfx/ImageFormats/CCITTDecoder.cpp

```diff
@@ -337,4 +347,32 @@ ErrorOr<ByteBuffer> decode_ccitt_rle(ReadonlyBytes bytes, u32 image_width, u32 i
     return decoded_bytes;
 }
 
+ErrorOr<ByteBuffer> decode_ccitt_group3(ReadonlyBytes bytes, u32 image_width, u32 image_height, Group3Options const& options)
+{
+    auto strip_stream = make<FixedMemoryStream>(bytes);
+    auto bit_stream = make<BigEndianInputBitStream>(MaybeOwned<Stream>(*strip_stream));
+
+    ByteBuffer decoded_bytes = TRY(ByteBuffer::create_zeroed(ceil_div(image_width * image_height, 8)));
+    auto output_stream = make<FixedMemoryStream>(decoded_bytes.bytes());
+    auto decoded_bits = make<BigEndianOutputBitStream>(MaybeOwned<Stream>(*output_stream));
+
+    if (options.dimensions == Group3Options::Mode::OneDimension) {
+        // 4.1.2 End-of-line (EOL)
+        // This code word follows each line of data. It is a unique code word that can never be found within a
+        // valid line of data; therefore, resynchronization after an error burst is possible.
+        // In addition, this signal will occur prior to the first data line of a page.
+        // ---
+        // NOTE: For whatever reason, the last EOL doesn't seem to be included
```

Comment:
very nit (completely up to you if you want to do it): I'd put this comment in `read_eol()` since that's where the code word is.

Reply:
I already tried both, and I think that it makes more sense to leave the quote next to where `read_eol` is used, as it's what the quote describes.